enhancements/scheduling/scheduler-profiles.md

```diff
@@ -0,0 +1,208 @@
+---
+title: scheduling-profiles
+authors:
+  - "@damemi"
+reviewers:
+  - "@maszulik"
+  - "@ingvagabund"
+  - "@deads2k"
+approvers:
+  - "@maszulik"
+  - "@ingvagabund"
+  - "@deads2k"
+creation-date: 2020-11-11
+last-updated: 2020-11-11
+status: provisional
+see-also:
+  - "https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/624-scheduling-framework/kep.yaml"
+  - "/enhancements/kube-apiserver/audit-policy.md"
+replaces:
+superseded-by:
+---
+
+# Scheduling Profiles
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a new field in the `schedulers.config.openshift.io/v1` API to allow
+configuration of enabled and disabled scheduler plugins via certain defined "profiles".
+
+This will allow users to configure pod scheduling beyond the default set of enabled plugins. It
+is effectively the equivalent to current scheduler Policy config, which has been deprecated.
```

Comment:
The above statement is not quite correct. You can say that it will simplify but it will definitely not allow more flexibility, which is how I understand above statement.